.github/workflows/dlp-pipelines.yml

```diff
@@ -10,6 +10,7 @@ env:
   PROJECT_ID: "dlp-dataflow-deid-ci-392604"
   DATASET_ID: "demo_dataset"
   PARQUET_DATASET_ID: "parquet_results"
+  REID_DATASET_ID: "reid_results"
```

Comment:
Can the variable and dataset name be "REID_WITHOUT_QUERY_ID" and "reid_without_query_results" to avoid confusion between normal reid and this special case?